### PR TITLE
SAK-29442 LB_PEER_EVAL_RESULT_S Sequence is missing in 10.x and next.…

### PR DESCRIPTION
**Pull request by sinmsinm**
… Added to conversion scripts

### CODE REVIEW

**Comment by juanjmerono** [comment]
LGTM

File: reference/docs/conversion/sakai_10_6-10_7_oracle_conversion.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+-- SAK-29442 Sequence LB_PEER_EVAL_RESULT_S Missing
+CREATE SEQUENCE "LB_PEER_EVAL_RESULT_S" MINVALUE 1 MAXVALUE 999999999999999999999999999 INCREMENT BY 1 START WITH 1 CACHE 20 NOORDER NOCYCLE;
```
Comment (ottenhoff) [inline reference/docs/conversion/sakai_10_6-10_7_oracle_conversion.sql]: It seems like this statement should be more minimal. Are the CACHE and NOORDER NOCYCLE necessary?

**Comment by sinmsinm** [comment]
I'm really not an expert of ORACLE Queries, but it was the one I got looking how was created one instance started with the auto.dll=true. I suppose that parameters are generated by hibernate or they are the default parameters. What would you prefer I do, do I clean the query?

**Comment by juanjmerono** [comment]
I think ```CREATE SEQUENCE LB_PEER_EVAL_RESULT_S;``` is the same, by default: 

- NOORDER
- NOCYCLE
- CACHE is 20
- MINVALUE is 1 
- MAXVALUE is 999999999999999999999999999
- INCREMENT BY is 1
- START WITH is 1

So, yes you could write a minimal statement in this case.

**Comment by sinmsinm** [comment]
I simplified the line.  I'll take account these advices for future scripts. 
Thanks.

**Pull request merged**